### PR TITLE
Contact.html finalizado y versión final 

### PR DESCRIPTION
**Pull request by LuisFlor7**

### CODE REVIEW

File: website/assets/css/style.css
Comment (ScarzoJ) [inline website/assets/css/style.css:1]: This change completely erases the previous styles work, are you sure you can do this here?

File: website/templates/contact.html [hunk 1]
```diff
@@ -4,21 +4,21 @@ <h2>Contact</h2>
...
             </div>
-            <div class="d-flex justify-content-end m-1">
+            <div class="d-flex justify-content-end m-2">
                 <button type="submit" class="btn btn-primary">Submit</button>
             </div>
         </div>
         </form>
-    </div>
+    </div>
 </div>
```
Comment (ScarzoJ) [inline website/templates/contact.html]: The ending of the file has some issues, the last two `div` tags are unnecessary and you should probably swap the `form` in line 22 with the `div` in line 21

**Review by ScarzoJ — CHANGES_REQUESTED** [review]

**Review by p3drac0 — APPROVED** [review]
Me voy a dormir contento con este trozo de código tan bonito

**Review by battanair — APPROVED** [review]
Está precioso

**Review by ScarzoJ — APPROVED** [review]

**Pull request merged**